tests/io/eager/test_delete.py

```diff
@@ -1,7 +1,8 @@
 import pytest
 
-from kartothek.io.eager import delete_dataset
+from kartothek.io.eager import delete_dataset, garbage_collect_dataset
 from kartothek.io.testing.delete import *  # noqa
+from kartothek.io.testing.gc import *  # noqa: F4
```

Comment:
There is already a `gc` test set up, see https://github.com/JDASoftwareGroup/kartothek/blob/master/tests/io/eager/test_gc.py

Please remove this again

Reply:
sure, make sense.